crates/holochain_cascade/src/authority/test.rs

```diff
@@ -126,6 +127,81 @@ async fn get_record() {
     assert_eq!(result, expected);
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn get_op_by_type() {
```

Comment:
Test the function and mainly the query that an authority runs to get the requested op from the DB.